openmdao/solvers/linear/direct.py

```diff
@@ -174,6 +174,9 @@ def _declare_options(self):
         self.options.undeclare("atol")
         self.options.undeclare("rtol")
 
+        # Always use an assembled jacobian.
```

Comment:
it's not 'Always', this is just setting the default, right?

Reply:
Made both requested edits.